Test_edge_creation

```diff
@@ -0,0 +1,126 @@
+var assert = require('assert');
+
+var self = this;
+
+describe('Issue: Found class name null or empty', function () {
```

Comment:
Please replace "Issue:" for "Bug #116:" so it will be listed among the other bugs in the build log, example: https://travis-ci.org/appscot/sails-orientdb/jobs/69971485#L1258